docs/pai-job-protocol.yaml

```diff
@@ -0,0 +1,223 @@
+%YAML 1.2
+---
+# OpenPAI Job Protocol YAML
+
+protocolVersion: String, required # Protocol version, current version is 2.
+name: String, required
+type: String, required # The type of the component. Must be one of the following: job, data, script, dockerimage, or output.
```

Comment:
Since this is the job protocol, should the type be only **job**?

Reply:
right, updated.